App.js

```diff
@@ -602,70 +641,53 @@ export default class App extends Component {
                 {(function() {
                   let rows = [];
                   for (let i = 0; i < item.emojiCount; i++) {
-                    console.log(item.emojiCount);
+                    //console.log(item.emojiCount);
+                    let emoji;
```

Comment:
let's be a little more descriptive with our Emoji variable names; we've been calling everything `emoji`

Reply:
Removed alot of the emoji stuff in my mergeChannels branch I have locally

Reply:
kk if you can make a little diagram of the payload after you've merged all the channels it'll be sick